app/views/coronavirus_landing_page/components/shared/_timeline.html.erb

```diff
@@ -1,23 +1,89 @@
 <div class="covid-timeline" data-module="track-links" data-track-category="pageElementInteraction" data-track-action="Timeline">
   <%= render "govuk_publishing_components/components/heading", {
-    text: timeline["heading"],
+    text: details.timeline["heading"],
     padding: true,
-    margin_bottom: 4
+    margin_bottom: 2
   } %>
 
-  <% if timeline["summary"] %>
-    <%= render_govspeak(timeline["summary"]) %>
-  <% end %>
+  <% if details.show_timeline_nations? %>
+    <%= form_tag coronavirus_landing_page_path,
+      method: :get,
+      class: "js-change-location" do |f| %>
+
+      <p class="govuk-body">
+        <%=t("coronavirus_landing_page.show.timeline.controls.status") %>
+        <span class="js-current-nation covid-timeline__nation"><%= details.selected_nation.titleize %></span>.
```

Comment:
I think [the comment](https://github.com/alphagov/collections/pull/2456#discussion_r663026960) I made on this last time got lost by the thread going on a tangent.

> Rather than having an incomplete sentence in the translation file it'd be good to move the HTML into there and call it as HTML. e.g:
> 
> ```erb
> <%=t("coronavirus_landing_page.show.timeline.controls.status_html", nation: @selected_nation.titleize) %>
> ```
> 
> ```yaml
>             status_html: Showing updates for <span class="js-current-nation covid-timeline__nation">%{nation}</span>.
> ```